supertokens_python/recipe/thirdpartyemailpassword/utils.py

```diff
@@ -114,11 +122,14 @@ def __init__(self,
                  email_verification_feature: ParentRecipeEmailVerificationConfig,
                  sign_up_feature: Union[InputSignUpFeature, None],
                  reset_password_using_token_feature: Union[InputResetPasswordUsingTokenFeature, None],
-                 override: OverrideConfig):
+                 get_email_delivery_config: Callable[[RecipeImplementation], EmailDeliveryConfigWithService[TypeThirdPartyEmailPasswordEmailDeliveryInput]],
```

Comment:
Should be of type RecipeInterface in the input

Reply:
Fixed. Added `EPRecipeInterface` too.